platform/view/services/storage/driver/sql/sqlite/binding.go

```diff
@@ -48,25 +56,31 @@ func (db *BindingStore) PutBinding(ctx context.Context, ephemeral, longTerm view
 	} else {
 		logger.DebugfContext(ctx, "Id [%s] is an unregistered long term ID", longTerm.UniqueID())
 	}
-	query := fmt.Sprintf(`
-		BEGIN;
-		INSERT INTO %s (ephemeral_hash, long_term_id)
-		VALUES ($1, $2)
-		ON CONFLICT DO NOTHING;
-		INSERT INTO %s (ephemeral_hash, long_term_id)
-		VALUES ($3, $4)
-		ON CONFLICT DO NOTHING;
-		COMMIT;
-		`, db.table, db.table)
 
-	logger.Debug(query, ephemeral.UniqueID(), longTerm.UniqueID())
-	_, err := db.writeDB.ExecContext(ctx, query, ephemeral.UniqueID(), longTerm, longTerm.UniqueID(), longTerm)
+	// Build single INSERT with multiple VALUES
+	query := fmt.Sprintf(`INSERT INTO %s (ephemeral_hash, long_term_id) VALUES `, db.table)
+
+	args := []interface{}{}
+	argsReferences := []string{"($1, $2)"}
+	args = append(args, longTerm.UniqueID(), longTerm)
+	for i, eph := range ephemeral {
+		args = append(args, eph.UniqueID(), longTerm)
+		oneArgRef := fmt.Sprintf("($%d, $%d)", i*2+3, i*2+4)
+		argsReferences = append(argsReferences, oneArgRef)
+	}
+
+	query += strings.Join(argsReferences, ", ")
+	query += " ON CONFLICT DO NOTHING;"
```

Comment:
we have the same query in the postgres impl; could we consider moving it to `sql/common/`?